.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug Report
+about: Report a bug encountered with the service-apis project
+labels: kind/bug
+
+---
+<!-- Please use this template while reporting a bug and provide as much info as possible.
+Not doing so may result in your bug not being addressed in a timely manner. Thank you!
+-->
+
```

Comment:
Commit hash (if applicable):

Reply:
Would this be used as a way to reference the change the introduced the bug?

Reply:
Yes -- right now that's basically the only thing to go on.
Later we can have people give platform, implementation etc.